.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,8 +22,16 @@ If applicable, add screenshots to help explain your problem.
 
 **Medusa (please complete the following information):**
  - OS: [e.g. ubuntu 14.04 x64]
- - Branch [e.g. master, develop]
- - Commit [e.g. c65ca2]
+ - Branch: [e.g. master, develop]
+ - Commit: [e.g. c65ca2]
+
+**Logs:**
```

Comment:
If you want you can add `<summary><b>Logs:</b></summary>` below `<details>`, it will show the title instead of `Details`. Edit my comment to see the code.

<details>
<summary><b>Logs:</b></summary>

```
<-- Please replace this whole line with your logs -->
```
</details>

Reply:
Yeah I figured it was easier that we move it down so users have very little chance of breaking the rendering above vs inside the list item.

Reply:
Users are bound to break our templates. If you remove the blank line between `<details>`/`<summary>` and the code block it would break markdown anyway... (open the logs)

<details>
<summary><b>Logs:</b></summary>
```
<-- Please replace this whole line with your logs -->
```
</details>

Reply:
- List item 1
- <details><summary><b>Logs:</b></summary>

  ```
  <-- Please replace this whole line with your logs -->
  ```
</details>

Reply:
Looks like your code works better, but sorry, I take it back.
The "Logs" should be more visible in the raw template since that's what users are seeing.
`<details><summary><b>Logs</b></summary>`
vs.
`**Logs:**`

Much easier to understand the latter, from the users' POV.

Reply:
So what're we doing here? I'm lost.. 😞

Reply:
Nothing.
I approved.